contrib/bin/stopKube

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+while true ; do
```

Comment:
I don't really know of a good way to stop hyperkube so this script just kills call docker containers that use a gcr.io/google_containers image name. Is there a better way to stop hyperkube and all of the containers it starts?